### PR TITLE
Comment out cryptic debug string

### PR DESCRIPTION
**Pull request by impaktor**
@fluffyfreak I didn't know if I should remove or keep the string, but I'll comment it out, since I'm getting warnings.

Possibly it should be used in an assert?

### CODE REVIEW

**Comment by fluffyfreak** [comment]
@impaktor That's not a debug string, that's the actual value assignment.
If you comment it out then the function doesn't do anything.

File: src/utils.cpp [hunk 1]
```diff
@@ -282,7 +282,7 @@ std::string DoubleToStr(double val)
 	static_assert(sizeof(double) == 8, "double isn't 8 bytes");
 	fu64 uval(val);
 	char str[128];
-	const int amt = sprintf(str, "%" PRIu64, uval.u);
+	//const int amt = sprintf(str, "%" PRIu64, uval.u);
```
Comment (fluffyfreak) [inline src/utils.cpp]: As above.

**Comment by fluffyfreak** [comment]
What is the warning that you're getting? The `static_assert` should catch mismatched sizes at compile time but it might be a GCC vs VS2015 issue where the type needs to be different on GCC instead of `PRIu64` or `PRIu32`.

**Comment by impaktor** [comment]
`amt` unused variable. And I can see why it would think that, since I also thought it. But sure, I haven't looked deeper into the code, so you're probably right.

**Comment by fluffyfreak** [comment]
ah amt is unused, in the other functions it's used for an assert to check that it hasn't overwritten the string, but in those two functions I clearly forgot to add that assert

**Comment by impaktor** [comment]
I'll close this, and forget it ever hapened.

**Pull request closed without merging**